pom.xml

```diff
@@ -127,11 +107,23 @@
             </exclusions>
         </dependency>
 
-        <!-- This is needed if the mongodb-springdata-v2-driver is being used -->
+        <dependency>
+            <groupId>org.mongodb</groupId>
+            <artifactId>mongodb-driver-sync</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <!-- This is needed if the mongodb-springdata-v3-driver is being used -->
```

Comment:
@chrisrohr Remind me why this has `test` scope rather than `provided`?

Reply:
Because it is not a compile time dependency.  I am creating a real client to perform the tests